lib/puppet/parser/functions/validate_rd_policy.rb

```diff
@@ -1,23 +1,30 @@
 require 'puppet/util/rundeck_acl'
 
-module Puppet::Parser::Functions
+# Validates the rundeck ACL policies
+# Usage:
+# Example:
+module Puppet
+  # Parser
+  module Parser
+    # Functions
+    module Functions
+      newfunction(:validate_rd_policy, :doc => <<-'ENDHEREDOC') do |args|
```

Comment:
aeehmpf.. please don't do this :O

what you *actually* should be doing is:


```ruby
Puppet::Parser::Functions.newfunction(:validate_rd_policy,…
```
thereby wholy removing one indentation level, rather than introducing three ;)